assets/chat/js/hashlinkconverter.js

```diff
@@ -45,13 +46,19 @@ class HashLinkConverter {
           return `#youtube/${match[1]}`;
         }
         videoId = url.searchParams.get('v');
+        timeStamp = url.searchParams.get('amp;t') || url.searchParams.get('t');
```

Comment:
Can you show me an example URL with a timestamp that `url.searchParams.get('amp;t')` catches? Is it something that shows up in the wild often enough that it has to be supported?

Reply:
So I haven't looked into it further but any normal youtube url comes back as amp;t for these searchParams once it's urlencoded. so this was the original commit I added the 't' to support the standard strings used in the unit tests for consistency and if we ever see a non url encoded url, but we could drop 't' support

Reply:
the shortened youtube urls don't because they don't have an ampersand as it's the first get param

Reply:
The real issue here is that HTML entities are escaped before the URL is converted to a hash link (and normalized). The appropriate moment to escape is just before it's rendered, after all preprocessing is done. Would you mind fixing that instead and reverting this line to just `url.searchParams.get('t')`?

Reply:
will do

Reply:
Using the decodedURL in the UrlFormatter takes care of this nicely by the looks of it, was there anything additional you were thinking we needed to do?